pyproject.toml

```diff
@@ -71,6 +71,8 @@ botocore = "1.31.72"
 boto3 = "1.28.72"
 requests = "^2.32.3"
 certifi = "^2023.7.22"
+regex = "2024.5.15"
```

Comment:
This because stdlib `re` does not support reverse search (`regex.REVERSE`) up to Python 3.12